src/ui/FileViewer/index.tsx

```diff
@@ -38,106 +39,108 @@ export const FileViewerComponent = (props: FileViewerComponentProps): ReactEleme
   const { stringSet } = useContext(LocalizationContext);
 
   return (
-    <div
-      className="sendbird-fileviewer"
-      data-testid="sendbird-fileviewer"
-      onKeyDown={onKeyDown}
-      // to focus
-      tabIndex={1}
-      ref={ref}
-    >
-      <div className="sendbird-fileviewer__header">
-        <div className="sendbird-fileviewer__header__left">
-          <div className="sendbird-fileviewer__header__left__avatar">
-            <Avatar height="32px" width="32px" src={profileUrl} />
+    <Modal>
```

Comment:
Just added `<Modal />`, no more change

Reply:
@HoonBaek In the ticket, I see no bug but what is the issue with not having Modal here?

Reply:
@liamcho 
The customer is using UIKit in a mobile environment, and on Android phones, there is a back button. To navigate back to the previous page using the back button, it is necessary to store the history for each page. We have supported this by allowing the customer to manage history through the `onMounted` event. The customer has requested that the same back button functionality, using history, be supported when navigating back from the `FileViewer` to the `Channel`.

Reply:
i got it. its for
```
  modal?: {
    onMounted?(params: { id: string; close(): void; }): void | (() => void);
  };
```
thanks!

Reply:
oh I understand the case now, thanks for explaining @HoonBaek :)